gdk4/src/cairo_interaction.rs

```diff
@@ -84,27 +57,48 @@ impl GdkCairoContextExt for Context {
         );
     }
 
+    #[doc(alias = "gdk_cairo_set_source_rgba")]
+    #[doc(alias = "set_source_rgba")]
     fn set_source_color(&self, rgba: &RGBA) {
         unsafe {
             ffi::gdk_cairo_set_source_rgba(self.to_glib_none().0, rgba.to_glib_none().0);
         }
     }
 
+    #[doc(alias = "gdk_cairo_set_source_pixbuf")]
     fn set_source_pixbuf(&self, pixbuf: &Pixbuf, x: f64, y: f64) {
         unsafe {
             ffi::gdk_cairo_set_source_pixbuf(self.to_glib_none().0, pixbuf.to_glib_none().0, x, y);
         }
     }
 
+    #[doc(alias = "gdk_cairo_rectangle")]
     fn add_rectangle(&self, rectangle: &Rectangle) {
         unsafe {
             ffi::gdk_cairo_rectangle(self.to_glib_none().0, rectangle.to_glib_none().0);
         }
     }
 
+    #[doc(alias = "gdk_cairo_region")]
     fn add_region(&self, region: &Region) {
         unsafe {
             ffi::gdk_cairo_region(self.to_glib_none().0, region.to_glib_none().0);
         }
     }
 }
+
+impl GdkCairoContextExt for Context {}
+
+mod sealed {
+    use cairo::Context;
+
+    pub trait Sealed {
```

Comment:
Is there any reason gdk4 doesn't enable the `use_glib` feature of cairo? This would allow rewriting this as `pub trait Sealed: IntoGlibPtr<*mut cairo::cairo_t>` I think

Reply:
Nevermind

Reply:
Tbh, i do believe we should enable that. Just no one ever thought of doing so :)

Reply:
Oh, then I enabled it in 75dfa9e4b32a25fe70587dfd94b36bfb9c97eb84 :)